coordinator/tributary/src/tests/blockchain.rs

```diff
@@ -52,10 +53,7 @@ fn block_addition() {
   assert!(blockchain.add_block::<N>(&block, vec![], validators).is_ok());
   assert_eq!(blockchain.tip(), block.hash());
   assert_eq!(blockchain.block_number(), 1);
-  assert_eq!(
-    Blockchain::<MemDb, SignedTransaction>::block_after(&db, genesis, &block.parent()).unwrap(),
-    block.hash()
-  );
+  assert_eq!(BlockAfterDb::get(&db, genesis, block.parent()).unwrap(), block.hash());
```

Comment:
Hm. The public API of this grew from a read-only accessor to full read-write access, which can likely break safety (except `block_after` was prior `pub(crate)`, so it doesn't actually since this API is also still only `pub(crate)`).

There's nothing to do in response to this comment, just being mindful and raising it as something to keep in mind with future edits which may actually affect the public API.